src/controller/CHIPDeviceController.cpp

```diff
@@ -1411,10 +1411,8 @@ void DeviceCommissioner::HandleAttestationResult(CHIP_ERROR err)
     }
     else
     {
-        // Handle error, and notify session failure to the commissioner application.
+        // Log an error message (no further handling currently)
         ChipLogError(Controller, "Failed to validate the Attestation Information");
```

Comment:
This is not enough: the flow to send the CSR request will not be done and commissioning will hang. Suggest replacing the large if/else with a first step `if(err != CHIP_NO_ERROR)` that does the logging and has the TODO, and the rest of the code out of any `if` as a common path.

Reply:
will do